src/admin-views/settings/sidebars/default-sidebar.php

```diff
@@ -45,81 +51,104 @@
 		)
 	)
 );
-$hero_section->set_title( new Heading( __( 'Finding and extending your tickets', 'event-tickets' ), 2, new Classes( 'tec-settings-form__sidebar-header' ) ) );
 
-$hero_section->add_section(
-	( new Settings_Section() )
-		->add_elements(
-			[
-				( new Paragraph() )->add_child(
-					new Plain_Text( __( 'Looking for additional functionality including recurring events, custom meta, community events, ticket sales, and more?', 'event-tickets' ) )
-				),
-				new Link(
-					'https://evnt.is/products',
-					__( 'Check out the available add-ons.', 'event-tickets' ),
-					null,
-					$external_attributes
-				),
-			]
-		)
+// Set Sidebar Title.
+$hero_section->set_title(
+	new Heading(
+		__( 'Supercharging your tickets', 'event-tickets' ),
+		2,
+		new Classes( 'tec-settings-form__sidebar-header' )
+	)
 );
 
-$hero_section->add_section(
-	( new Settings_Section() )
-		->set_title( new Heading( __( 'Documentation', 'event-tickets' ), 3 ) )
-		->add_elements(
-			[
-				new Link(
-					'https://evnt.is/1apn',
-					__( 'Getting started guide', 'event-tickets' ),
-					null,
-					$external_attributes
-				),
-				$break,
-				new Link(
-					'https://evnt.is/1bbw',
-					__( 'Knowledgebase', 'event-tickets' ),
-					null,
-					$external_attributes
-				),
-			]
-		)
+// Build Beginner Resources List.
+$beginner_list = new Unordered_List(
+	new Classes( 'tec-tickets__admin-banner-kb-list' )
```

Comment:
Is this a style we'll reuse? Do we want a class that is more generic?

Reply:
Right now I put a class on the element. However, there aren't any styles created yet. Would we reuse the list of resources in other sections?